theme/layout/z-indexes.less

```diff
@@ -0,0 +1,5 @@
+@z-index-header-overlay: 4;
```

Comment:
Yep it definitely is, I would even go for 1xx, but I'm conscious of moving Chalice to these variables, in this case it will be less painful if we miss one or two. But yeah, probably should be just careful in migrating...thoughts?